docs/SRIOV.md

```diff
@@ -0,0 +1,260 @@
+
+### Kernel IOMMU Support
+
+Using **dmesg**, you can verify if IOMMU is enabled with the following command:
+
+```
+# dmesg | grep IOMMU
+```
+
+If you do not see the message ```DMAR: IOMMU enabled```, then proceed with the following steps:
+
+First, enable IOMMU support in the kernel by modifying the GRUB configuration at **/etc/default/grub**:
+
+```
+GRUB_CMDLINE_LINUX="... intel_iommu=on" #Intel-based Systems
+
+GRUB_CMDLINE_LINUX="... amd_iommu=on" #AMD-based Systems
+```
+
+Next, update GRUB:
+
+```
+update-grub
+```
+
+> Once the kernel configuration has been modified, you must reboot for the changes to take effect.
+
+### IOMMU Passthrough Support
+
+To enable IOMMU passthrough support in the kernel, please complete the following steps:
+
+First, enable passthrough support in the kernel by modifying the GRUB configuration at **/etc/default/grub**:
+
+```
+GRUB_CMDLINE_LINUX="... iommu=pt"
+```
+
+Then, update GRUB:
+
+```
+update-grub
+```
+
+> Once the kernel configuration has been modified, you must reboot for the changes to take effect.
+
+## Deploying PMO with SR-IOV support using Express
+
+Using the Platform9 Express tool, operators can deploy PMO with support for SR-IOV. The Express tool will perform many of the tasks outlined in the previous sections, including enabling IOMMU and passthrough support in the kernel, as well as implementing a unit file for persisting VFs across reboots.
+
+> Given the complexity involved in supporting Mellanox NICs, the Express tool will initially only support Intel NICs using the ixgbe driver. This includes the Intel X520, X540, and X550 families.
+
+The necessary configuration details can be implemented globally using **group_vars**, or on an individual host basis using **host_vars**. Each method is described below.
+
+### Host Variables
+
+Compute node-specific configurations can be implemented using what is known as host_vars. Configurations that may vary between hosts include:
+
+* Network interface name
+* Quantity of network interfaces used for SRIOV
+* Provider network mappings
+* Number of VFs per interface
+ 
+Using **host_vars**, the following are some variables that can be modified:
+
+* physical_device_mappings (required)
+* sriov_numvfs (required)
+* neutron_ovs_bridge_mappings (optional)
+
+In this example, two hosts have different NICs installed that report different names to the operating system.
+
+```
+root@compute01:~# ip link show
+...
+6: ens1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq portid 0002c90300ffe511 state UP mode DEFAULT group default qlen 1000
+    link/ether 00:02:c9:ff:e5:10 brd ff:ff:ff:ff:ff:ff
+7: ens1d1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq portid 0002c90300ffe512 state UP mode DEFAULT group default qlen 1000
+    link/ether 00:02:c9:ff:e5:11 brd ff:ff:ff:ff:ff:ff
+```
+
+```
+root@compute02:~# ip link show
```

Comment:
Rename to hv02 to align with readme / sample inventory file

Reply:
Fixed.